openvm/src/powdr_extension/vm.rs

```diff
@@ -82,18 +84,35 @@ impl<F> PowdrExtension<F> {
             implementation,
             bus_map,
             airs,
+            _marker: std::marker::PhantomData,
         }
     }
 }
 
 #[derive(ChipUsageGetter, From, AnyEnum, InstructionExecutor, Chip)]
 #[allow(clippy::large_enum_variant)]
-pub enum PowdrExecutor<F: PrimeField32> {
-    Powdr(PowdrChip<F>),
-    Plonk(PlonkChip<F>),
+pub enum PowdrExecutor<
+    F: PrimeField32,
+    A: Adapter<
+            Field = F,
+            Instruction = Instr<F>,
+            InstructionHandler = OriginalAirs<F>,
+            AirId = String,
+        > + 'static,
```

Comment:
Usually the associated type bounds are not required on data structure definitions (`PrimeField32` is because of the derives expect it iirc)